### ISSUE TITLE
Explain parry and accuracy better in the future

### ISSUE THREAD

**Issue by Hejnewar** [wesnoth/wesnoth#5860]
### Game and System Information
 - 1.15.13

### Describe the desired feature
Parry and accuracy should be better explained in game. Currently just % is displayed and in tooltip name, there is no information about what they do. 

### Screenshots
![parry](https://user-images.githubusercontent.com/41817546/121820739-be029b00-cc94-11eb-9acd-bac94d29d462.JPG)
![parry1](https://user-images.githubusercontent.com/41817546/121820743-bfcc5e80-cc94-11eb-816b-e18478b7f9e2.JPG)

**Comment by Wedge009** [comment]
I haven't seen parry/accuracy before - is it new in 1.15?

**Comment by soliton-** [comment]
Already there since a couple of stable versions but not used in mainline.

**Comment by Wedge009** [comment]
Yeah, I realised that afterwards, I think while looking at #4348.

**Comment by stevecotton** [comment]
Adding "bonus to chance to hit" to the tooltip in reports.cpp would help. A small task, but it also requires cleaning up the code to make it better to translate, allowing the translator to move the number to the best place in the string, maybe having separate strings for bonus vs malus.

**Comment by cooljeanius** [comment]
Related: #7895 for `u.w.o`, and #2517 for the in-game component
